js/script.js

```diff
@@ -1,8 +1,15 @@
 $(function () {
     $(document).scroll(function () {
-        var $nav = $(".fixed-top");
-        $nav.toggleClass('scrolled', $(this).scrollTop() > $nav.height());
-        var $text = $(".text-white");
-        $text.toggleClass('dark', $(this).scrollTop() > $nav.height())
+        var $nav = $("#nav-bar");
+        var $navSupportedContent = $("#navbarSupportedContent");
+        var isShown = $navSupportedContent.attr("class").includes(" show");
+        if(!isShown)
+            $nav.toggleClass("scrolled", $(this).scrollTop() > $nav.height());
+    });
+    $("#toggler").click(() => {
+        var $nav = $("#nav-bar");
```

Comment:
Why redefine twice? Should we make it global?

Reply:
yes will be better